src/main/java/net/earthcomputer/clientcommands/command/AreaStatsCommand.java

```diff
@@ -0,0 +1,205 @@
+
+            chunks++;
+            blocks = loop(minX, (minX >> 4) * 16 + 15, minZ, (minZ >> 4) * 16 + 15, minY, maxY, blocks, blockPredicate, minMinChunk, mutablePos);
+
+            chunks++;
+            blocks = loop(minX, (minX >> 4) * 16 + 15, (maxZ >> 4) * 16, maxZ, minY, maxY, blocks, blockPredicate, minMaxChunk, mutablePos);
+
+            chunks++;
+            blocks = loop((maxX >> 4) * 16, maxX, minZ, (minZ >> 4) * 16 + 15, minY, maxY, blocks, blockPredicate, maxMinChunk, mutablePos);
+
+            chunks++;
+            blocks = loop((maxX >> 4) * 16, maxX, (maxZ >> 4) * 16, maxZ, minY, maxY, blocks, blockPredicate, maxMaxChunk, mutablePos);
+
+            for (int minMinMaxMin = minMinChunk.getPos().x + 1; minMinMaxMin < maxMinChunk.getPos().x; minMinMaxMin++) {
+                chunks++;
+                final WorldChunk chunk = world.getChunk(minMinMaxMin, minMinChunk.getPos().z);
+                blocks = loop(16 * minMinMaxMin, 16 * minMinMaxMin + 15, minZ, (minZ >> 4) * 16 + 15, minY, maxY, blocks, blockPredicate, chunk, mutablePos);
+            }
+            for (int minMinMinMax = minMinChunk.getPos().z + 1; minMinMinMax < minMaxChunk.getPos().z; minMinMinMax++) {
+                chunks++;
+                final WorldChunk chunk = world.getChunk(minMinChunk.getPos().x, minMinMinMax);
+                blocks = loop(minX, (minX >> 4) * 16 + 15, 16 * minMinMinMax, 16 * minMinMinMax + 15, minY, maxY, blocks, blockPredicate, chunk, mutablePos);
+            }
+            for (int minMaxMaxMax = minMaxChunk.getPos().x + 1; minMaxMaxMax < maxMaxChunk.getPos().x; minMaxMaxMax++) {
+                chunks++;
+                final WorldChunk chunk = world.getChunk(minMaxMaxMax, minMaxChunk.getPos().z);
+                blocks = loop(16 * minMaxMaxMax, 16 * minMaxMaxMax + 15, (maxZ >> 4) * 16, maxZ, minY, maxY, blocks, blockPredicate, chunk, mutablePos);
+            }
+            for (int maxMinMaxMax = maxMinChunk.getPos().z + 1; maxMinMaxMax < maxMaxChunk.getPos().z; maxMinMaxMax++) {
+                chunks++;
+                final WorldChunk chunk = world.getChunk(maxMinChunk.getPos().x, maxMinMaxMax);
+                blocks = loop((maxX >> 4) * 16, maxX, 16 * maxMinMaxMax, 16 * maxMinMaxMax + 15, minY, maxY, blocks, blockPredicate, chunk, mutablePos);
+            }
+            for (int chunkX = minMinChunk.getPos().x + 1; chunkX < maxMinChunk.getPos().x; chunkX++) {
+                for (int chunkZ = minMinChunk.getPos().z + 1; chunkZ < minMaxChunk.getPos().z; chunkZ++) {
+                    chunks++;
+                    final WorldChunk chunk = world.getChunk(chunkX, chunkZ);
+                    blocks = loop(16 * chunkX, 16 * chunkX + 15, 16 * chunkZ, 16 * chunkZ + 15, minY, maxY, blocks, blockPredicate, chunk, mutablePos);
+                }
+            }
+        }
+
+        final long entities = StreamSupport.stream(world.getEntities().spliterator(), false)
```

Comment:
Guava has a nice method:
```suggestion
        final long entities = Streams.stream(world.getEntities())
```

Reply:
Oh, nice. I did notice it was annotated as `@Beta`, though. Fine by me.